docs/developer/architecture/logs.rst

```diff
@@ -197,6 +197,444 @@ overview page.
+  metadata:
+    name: metalk8s-fluent-bit-config
+    namespace: metalk8s-logging
+  data:
+    config.yaml: |-
+      apiVersion: addons.metalk8s.scality.com
+      kind: FluentBitConfig
+      spec:
+        output: |-
+          [OUTPUT]
+            Name            es
+            Match           *
+            Host            localhost
+            Port            9200
+            Logstash_Format On
+            Replace_Dots    On
+            Retry_Limit     False
+
+.. note::
+
+  Fluent Bit configuration is not a YAML file and looks more like an INI one.
+  CSC is not able to take care of these files and even if we add support for
+  this format, it will be hard to implement the merging with defaults mechanism
+  as blocks can not really be identified, because there is no unique name or ID.
+  Since we only need to provide to the user the possiblity to add a new output,
+  it should be sufficient as we don't need to edit what's deployed by MetalK8s.
+
+Loki
+""""
+
+Loki's configuration is stored as a Secret, as for Fluent Bit, we need
+to expose few parameters to the user for customization.
+We will use the CSC mechanism to provide the interface for customization,
+with a ConfigMap ``metalk8s-loki-config`` in ``metalk8s-logging`` Namespace.
+
+A Salt module will be developed to manipulate this object, so the logic
+can be kept in only one place.
+
+This module must provide necessary methods to show or edit the configuration
+in 2 different ways:
+
+   * simple
+   * advanced
+
+The ``simple`` mode will only display and allow to change some specific
+configuration, such as the retention or the number of replicas, and in an as
+simple as possible manner for the user.
+
+The ``advanced`` mode will allow to change all the configuration points,
+exposing the whole configuration as a plain YAML.
+
+This module will then be exposed through a CLI and a UI.
+
+Fluentd
+"""""""
+
+Monitoring
+~~~~~~~~~~
+
+Prometheus
+""""""""""
+
+To monitor every services in our log centralization stack, we will need to
+deploy ``ServiceMonitor`` object and expose the ``/metrics`` route of all
+these components.
+It will allow Prometheus Operator to configure Prometheus and automatically
+start scraping these services.
+This can be achieved by adding the following configuration in the helm
+chart values:
+
+For Fluent Bit, in ``charts/loki-stack.yaml``:
+
+.. code-block:: yaml
+
+  fluent-bit:
+    serviceMonitor:
+      enabled: true
+      additionalLabels:
+        release: "prometheus-operator"
+
+For Loki, in ``charts/loki-stack.yaml``:
+
+.. code-block:: yaml
+
+  loki:
+    serviceMonitor:
+      enabled: true
+      additionalLabels:
+        release: "prometheus-operator"
+
+For Fluentd, in ``charts/fluentd.yaml``:
+
+.. code-block:: yaml
+
+  metrics:
+    serviceMonitor:
+      enabled: true
+      additionalLabels:
+        release: "prometheus-operator"
+
+We also need to define alert rules based on metrics exposed by these services.
```

Comment:
what would be the formula to compute Log centralization Health?

Reply:
It's written, we need to define this formula.
We could have a basic formula which only make sure that every pods (or at least the daemonset ones & 1 per deployment) in the metalk8s-logging namespace is healthy, what do you think ?

Reply:
Suggestion from @xaf-scality was to also consider the whole pipeline: can we use alerting rules to compare the amount of logs seen by collectors with what gets ingested in Loki? Would help us see if something in the pipeline is broken, meaning we'd be losing logs.